### DISCUSSION TITLE
Request change to France 2 IPTV link

### DISCUSSION THREAD

**Discussion by Nic787** [iptv-org/community#1121] [Q&A]
Hi,

I finally tested France 2 with a VPN and I saw that it use a link with closed captioning/audio transcription. Is it possible find a clean link for France 2?

thank you

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
